simulariumio/data_objects/trajectory_data.py

```diff
@@ -38,9 +37,9 @@ def __init__(
 
         Parameters
         ----------
-        box_size : np.ndarray (shape = [3])
-            A numpy ndarray containing the XYZ dimensions
-            of the simulation bounding volume
+        meta_data : MetaData
```

Comment:
Doesn't it seem odd to have `time_units` and `spatial_units` separate from `meta_data`....... since they are technically metadata...?  Not that I _want_ to fiddle with changes to the unit data shape in simularium-viewer again... 🙈

Reply:
I wonder if it might be appropriate to change `meta_data`/`MetaData` to `scene_data`/`SceneData` or something like that? Metadata is very broad. But if I'm the only person who thinks this, don't worry about it haha

Reply:
some packages don't need the user to specify units, for example Cytosim is always seconds and micrometers, and this metadata object is used by all converters. I think it would be confusing if a Cytosim user thought they could enter units but then they were overridden by the Cytosim converter. And it seems fine to have units as fields on TrajectoryData?

Reply:
Oh I see, okay.